lib/arel_toolkit/version.rb

```diff
@@ -1,3 +1,3 @@
 module ArelToolkit
-  VERSION = '0.5.0'.freeze
+  VERSION = '0.6.0'.freeze
```

Comment:
This will be bumped when releasing a new version, no need!

Reply:
:+1: i'll revert this back to 0.5.0 and merge, the changelog data can stay as i doubt the automated changelog will be as obvious due to the fact my commits won't squash nicely 🗡️